OrbitGl/PickingManager.h

```diff
@@ -56,11 +56,19 @@ struct PickingID {
     return id;
   }
   static Color GetColor(Type a_Type, uint32_t a_ID, BatcherId batcher_id = TIME_GRAPH) {
+    static_assert(sizeof(PickingID) == sizeof(Color),
+      "PickingId and Color must have the same size");
     PickingID id = Get(a_Type, a_ID, batcher_id);
-    return *(reinterpret_cast<Color*>(&id));
+    Color color;
+    memcpy(&color, &id, sizeof(PickingID));
```

Comment:
I think we should use `std::memcpy` and include `<cstring>`.

Reply:
Done